### PR TITLE
166266848 Deploy app to heroku

### PR DESCRIPTION
**Pull request by sebbss**
#### What does this PR do?

Deploy the app on heroku

#### Description of Task to be completed?

- created a Procfile to specify the commands executed by the app on startup
- created an app.json file that  manifests a format for describing web apps

#### How should this be manually tested?

- follow the link to the deployed app [heroku link](https://wger-space.herokuapp.com/en/software/features)

#### What are the relevant pivotal tracker stories?

[#166266848](https://www.pivotaltracker.com/story/show/166266848)

#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by richien — CHANGES_REQUESTED** [review]
Hi @sebbss, first of all, thanks for finally getting the app to deploy. I was wondering, is this the final display or are you working on fixing the issues with rendering CSS and JS in the site?

**Review by NicholusMuwonge — CHANGES_REQUESTED** [review]
@sebbss this Took you some hard work, I thank you for your effort, The application link works fine on my end, though I was wondering If you were supposed to create two links for both production and staging?

**Comment by frankopkusianwar** [comment]
> @sebbss this Took you some hard work, I thank you for your effort, The application link works fine on my end, though I was wondering If you were supposed to create two links for both production and staging?

@NicholusMuwonge I think @sebbss should add the capability to use review apps such that temporary deployment links are created whenever a PR is created

File: app.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    "buildpacks": [
+      {
+        "url": "heroku/nodejs"
+      },
+      {
+        "url": "heroku/python"
+      }
+    ],
+    "stack": "heroku-18"
+  }
```
Comment (nakatuddesuzan) [inline app.json]: Hey @sebbss , kindly add a line below here

File: wger/settings_global.py [hunk 2]
```diff
@@ -370,3 +372,14 @@
...
+#
+#STATIC FILES
+#
+PROJECT_ROOT = os.path.join(os.path.abspath(__file__))
+STATIC_ROOT = os.path.join(PROJECT_ROOT, 'staticfiles')
+STATIC_URL = '/static/'
+STATICFILES_DIRS = (
+    os.path.join(PROJECT_ROOT, "static"),
+)
+# STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```
Comment (nakatuddesuzan) [inline wger/settings_global.py]: Here too

**Review by richien — APPROVED** [review]

**Comment by nakatuddesuzan** [comment]
> @sebbss this Took you some hard work, I thank you for your effort, The application link works fine on my end, though I was wondering If you were supposed to create two links for both production and staging?

Lets do staging for the mean time

**Review by nakatuddesuzan — APPROVED** [review]

**Review by NicholusMuwonge — APPROVED** [review]

**Pull request merged**